src/main/java/springboot/Controller/PersonController.java

```diff
@@ -26,4 +30,12 @@ public String personListPage(Model model) {
         model.addAttribute("person", service.getAllPersons());
         return "personListPage";
     }
+
+    @PostMapping(value = "/selectUser")
```

Comment:
do it like this
public String setPersonAsUser (@RequestParam Integer user, Model model){
        model.addAttribute("user", service.getPerson(user));
        return "personListPage";